resources/assets/coffee/react/_components/beatmap-icon.coffee

```diff
@@ -37,7 +37,9 @@ el = React.createElement
 
     showTitle = @props.showTitle && !@props.overrideVersion?
 
+    mode = if beatmap.convert then 'osu' else beatmap.mode
```

Comment:
1. http://puu.sh/q9jlG.png
2. the old website always chose the highest difficulty (aka the last one) by default
3. the new design still doesn't account for the converts in any way, so right now i would have done pretty much the same (@arflyte poke poke)